manifests/Cromha Expansion.yaml

```diff
@@ -0,0 +1,14 @@
+name: Cromha Expansion Plugin
+authors: @OcelotWalrus/Cromha, @Lumbar527
+homepage: https://github.com/OcelotWalrus/Cromha-Expansion-plugin
+license: GPL-3.0-or-later
+version: v0.0.1
```

Comment:
This should probably be `v0.0.1-hotfix` to align with the tag name. That also means you can just use $version in the autoupdate, without the extra `-hotfix`, which would presumably break the moment you push 0.0.2.

Reply:
Done.